activeuuid.gemspec

```diff
@@ -21,6 +21,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency "database_cleaner"
   s.add_development_dependency "fabrication"
   s.add_development_dependency "forgery"
+  s.add_development_dependency "pry"
```

Comment:
FYI, pry-byebug adds both pry and byebug, so pry doesn't need to be declared as a separate dependency.

Reply:
Agree, thanks @jlw !